.github/workflows/generate-lists.yml

```diff
@@ -74,6 +74,38 @@ jobs:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         branch: lists
 
+  success-notification:
+    runs-on: ubuntu-latest
+    needs: generate-lists
+    if: success()
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: lists
+          fetch-depth: 0
+      - name: Get commit details
+        id: get-commit-details
+        run: |
+          COMMIT_MSG=$(git log -1 --pretty=format:"%s")
+          MODIFIED_FILES=$(git show --name-only --pretty=format: HEAD | grep -c "sanctioned_addresses_" || echo "0")
+          if [ "$MODIFIED_FILES" -gt "0" ]; then
+            echo "SLACK_MESSAGE=The OFAC sanctioned digital currency addresses list generation job completed successfully. $MODIFIED_FILES address files were updated." >> $GITHUB_OUTPUT
+          else
```

Comment:
Let's leave the else clause in for now; however, I imagine we may want to remove it in the future. My primary "ask" is knowing that the workflow found a change in the list of sanctioned addresses. (Since the workflow reports on failure, having the else clause for now is good for us to see that the `success` is running...)